proposal/COMP110 architecture essay/comp110 essay proposal.tex

```diff
@@ -0,0 +1,72 @@
+\documentclass{article}
+
+\title{Essay Proposal - COMP110 Computer Architecture Essay}
+
+\author{TE182503}
+
+\begin{document}
+
+\maketitle
+
+\section*{Topic}
+My essaye will be on Convexity-based collision detection for a 2D game engine.
+
+\section*{Paper 1}
+
+\begin{description}
+\item[title:] The Construction of a Predictive Collision 2D Game Engine
+\item[Citation:]\cite{<mulley2013construction>} 
```

Comment:
A technique is described, but it is more sophisticated that collision detection - in that it models the predicted time of collision of two objects with given velocities. Something a bit less sophisticated is recommended.